pkg/querier/queryrange/roundtrip.go

```diff
@@ -318,9 +318,7 @@ func NewDetectedLabelsCardinalityFilter(rt queryrangebase.Handler) queryrangebas
 			var result []*logproto.DetectedLabel
 
 			for _, dl := range resp.Response.DetectedLabels {
-				if dl.Cardinality > 1 && dl.Cardinality < 50 {
-					result = append(result, &logproto.DetectedLabel{Label: dl.Label, Cardinality: dl.Cardinality})
-				}
+				result = append(result, &logproto.DetectedLabel{Label: dl.Label, Cardinality: dl.Cardinality})
```

Comment:
I'm fine with this, though if it were a request parameter it would give us more flexibility to find the right balance in the future. But, let's nuke it for now and see if it becomes a problem.